bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -224,7 +224,9 @@ class ModelImporter(
                     val existingNodeReference = originalIdToExisting[expectedId]
                     if (existingNodeReference == null) {
                         val newChild = existingParent.addNewChild(role, newIndex, expectedConcept)
-                        newChild.setPropertyValue(NodeData.idPropertyKey, expectedId)
+                        if (newChild.originalId() == null) {
```

Comment:
nit: Can this be sensibly teste? Maybe in `bulk-model-sync-lib-mps-test`?

Reply:
This is covered by the new test case. That's how I discovered the issue.